vm/src/obj/objsequence.rs

```diff
@@ -14,7 +14,7 @@ pub trait PySliceableSequence {
     fn get_pos(&self, p: i32) -> usize {
         if p < 0 {
             if -p as usize > self.len() {
-                // return something that is out of bounds so get_item raises an IndexError
+                // return something that is out of bounds so `get_item` raises an IndexError
                 self.len() + 1
```

Comment:
Instead of using this trick here, it would be better to return `Result<usize, PyObjectRef>` instead of `usize` from this function and create the exception at this point in the code.

Reply:
Yes agreed. Sorry this change was from #297, I accidentally ref'd the issue instead of the PR